__tests__/report-portal-client.spec.js

```diff
@@ -1,7 +1,7 @@
 const process = require('process');
-const RPClient = require('../lib/report-portal-client');
-const helpers = require('../lib/helpers');
-const { OUTPUT_TYPES } = require('../lib/constants/outputs');
+const RPClient = require('../build/lib/report-portal-client').default;
+const helpers = require('../build/lib/helpers');
+const { OUTPUT_TYPES } = require('../build/lib/constants/outputs');
```

Comment:
I suppose the tests should test the sources, not built version.
So, please revert the imports.
Also test files should be also migrated to TypeScript.